### PR TITLE
Complete react-redux capstone project

### PR DESCRIPTION
**Pull request by serengia**
Hello. Kindly approve the request to merge `development` branch to `main` branch.
- The branch contains all the [requirements ](https://github.com/microverseinc/curriculum-react-redux/blob/main/capstone/react_capstone.md) and functionalities of the capstone project.
- You can also see my [live project here](https://covid-19-data-app.onrender.com/)

### CODE REVIEW

**Review by AshakaE — COMMENTED** [review]
### Hi @serengia  🙋 , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

# Invalid Code Review Request ❌ 

You have submitted a project, that lacks a video presentation for your project

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: src/pages/CountriesDataPage.js [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+        <GlobalStats />
+        {view === "countries" && (
+          <div className={s["countries-view"]}>
+            <ul className={s["countries-list"]}>
+              {filterCountries(countries).map((c) => (
+                <Country key={c.country} data={c} />
+              ))}
+            </ul>
+          </div>
+        )}
```
Comment (m05e5) [inline src/pages/CountriesDataPage.js:77]: - [ ] Hi you really did a fantastic job but as part of the requirements when we click on an item (country) its supposed to navigate to the country details page. please kindly add navigations to your country cards( I know you did it with continents, but that's a different flow).

**Review by m05e5 — CHANGES_REQUESTED** [review]
Hi @serengia , 

Good job so far. !
There are some issues that you still need to work on to go to the next project but you are almost there!
mario_ladders


## Highlights


## Required Changes ♻️
* You really did a great job but there are some features you need to improve

- [x] Hi as part of the requirements, you're supposed to attach to your readme file or the PR description the link to a 5mins max video you recorded in which you showcase your app and explain some interesting features you developed.

- [x] Please check the comments below 

_I'm sure you can do it just keep it up_

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

_@m05e5_

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by serengia** [comment]
Thank you for your feedback. @AshakaE  @m05e5 . I am facing some challenges but I will do my best to submit the changes as soon as possible.

**Review by abdulhamiid — APPROVED** [review]
Hi @serengia 👋, 

In my opinion, your project is ready for presentation!  There is nothing else to say other than... it's time to merge it :shipit: 

giphy

# Highlights
✅ No Linter error
✅ Correct use of Git flow
✅ Very descriptive Pull Request(PR)
✅ Neat Commit messages

Good luck! 🎉 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Comment by serengia** [comment]
Thank you.

**Pull request merged**